apps/portals/src/portal-components/DetailsPage/DetailsPage.tsx

```diff
@@ -341,7 +341,7 @@ export const SplitStringToComponent: React.FC<{
   if (overrideSqlSourceTable) {
     // use the search param value to override the sql param.
     injectedProps['sql'] = `SELECT  *  FROM  ${value}${
-      rowVersionNumber ? `.${rowVersionNumber}` : ''
+      rowVersionNumber && columnName == 'id' ? `.${rowVersionNumber}` : ''
     }`
   }
```

Comment:
Wait, why are we using the `rowVersionNumber` at all in this query? Shouldn't a version number, if any, come from the `value`?